cpp/models/ode_seair/parameters.h

```diff
@@ -0,0 +1,277 @@
+/* 
+* Copyright (C) 2020-2023 German Aerospace Center (DLR-SC),
+* Forschungszentrum Juelich (FZJ)
```

Comment:
```suggestion
* 2020-2024 MEmilio
```
please use just time horizon and MEmilio for generic reference.

Reply:
also in all other "new" files

Reply:
*done*